MAUI/CheckBox/Getting-Started.md

```diff
@@ -0,0 +1,356 @@
+
+## Handler registration 
+
+In the MauiProgram.cs file, register the handler for the Syncfusion core.
+
+{% highlight c# hl_lines="6 17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace ButtonSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}   
+
+{% endhighlight %}  
+
+## Create a Simple .NET MAUI SfCheckBox
+
+The [.NET MAUI CheckBox] (SfCheckBox) control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a [`SfCheckBox`] and configure its elements.
+
+## Add namespace for referred assemblies
+
+{% tabs %}
+{% highlight xaml %}
+
+    xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons"
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Buttons;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Refer SfCheckBox control with declared suffix name for Namespace
+
+{% tabs %}
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                 xmlns:local="clr-namespace:GettingStarted"
+	             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons"             
+	             x:Class="GettingStarted.MainPage">
+        <ContentPage.Content>
+            <StackLayout>
+                  <syncfusion:SfCheckBox x:Name="checkBox"/>
+            </StackLayout>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Buttons;
+
+    namespace GettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                StackLayout stackLayout = new StackLayout();
+                SfCheckBox checkBox = new SfCheckBox();
+                stackLayout.Children.Add(checkbox);
+                this.Content = stackLayout;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting caption
+
+The check box caption can be defined using the [`Text`] property of [`SfCheckBox`]. This caption normally describes the meaning of the check box and it displays next to check box.
```

Comment:
The check box caption can be defined using the "Text" property of "SfCheckBox". This caption typically describes the meaning of the check box and is displayed next to the check box.